src/main/scala/templatecontrol/Task.scala

```diff
@@ -50,6 +50,9 @@ class CopyTask(c: Config) extends Task {
       val source: File = findTemplate(c.template)
       val dest: File = file"${workingDir.path.toAbsolutePath}${c.path}"
       blocking {
+        // create file if non-existent
+        if (dest.notExists) dest.createFile()
```

Comment:
the copy was only working if the file already existed

Reply:
Does `createFile` when `dest` is a relative path with non-existent folder and a file?

E.g.: when trying to `copy` `./src/main/resources/logback.xml` with template control.